### ISSUE TITLE
Column names for show tables are wrong

### ISSUE THREAD

**Issue by systay** [vitessio/vitess#7253]
#### Overview of the Issue

When doing `show tables` on a database named `foobar`, the column in the result set has `vt_` prepended:

```sql
mysql> use foobar;
Database changed
mysql> show tables;
+---------------------+
| Tables_in_vt_foobar |
+---------------------+
| music               |
| user                |
| user_extra          |
+---------------------+
3 rows in set (0.00 sec)
```

**Comment by deepthi** [comment]
database -> keyspace?
It seems that the column name is coming from the underlying database name which defaults to `vt_foobar`. If it were overridden using `-init_db_name_override` flag to `xyz` then the column name would be `Tables_in_xyz`. Are we saying that it should always be `Tables_in_foobar` regardless of underlying database name?

**Comment by systay** [comment]
Yeah, this is all about imitating MySQL behaviour. The issue is not critical per se, but the behaviour is surprising, wouldn't you agree?
